src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private final Random random = new Random();
+
+    public String getRandomColor() {
+        String[] colors =
+                {"brown", "yellow", "red", "pink", "white", "black", "orange", "violet", "blue"};
+        return colors[random.nextInt(colors.length)];
```

Comment:
Create enum for Color
```suggestion
        int index = random.nextInt(Color.values().length);
        return Color.values()[index].name();
```